charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "topolvm.fullname" . }}-controller
+  labels:
+  {{- include "topolvm.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.controller.replicaCount }}
+  minReadySeconds: {{ .Values.controller.minReadySeconds }}
+  strategy:
+{{ toYaml .Values.controller.updateStrategy | indent 4 }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "topolvm.fullname" . }}-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "topolvm.fullname" . }}-controller
+    spec:
+      terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
+      {{- with .Values.controller.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      securityContext:
+{{ toYaml .Values.securityContext | indent 8 }}
+      serviceAccountName: {{ template "topolvm.fullname" . }}-controller
+      containers:
+        - name: topolvm-controller
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command:
+            - /topolvm-controller
+            - --cert-dir=/certs
+          ports:
+            - containerPort: 9808
+              name: healthz
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /metrics
+              port: 8080
+              scheme: HTTP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: healthz
+            failureThreshold: 3
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+            periodSeconds: 60
+          resources:
+{{ toYaml .Values.controller.resources | indent 12 }}
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /run/topolvm
+            - name: certs
+              mountPath: /certs
+        - name: csi-provisioner
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command:
+            - /csi-provisioner
+            - "--csi-address=/run/topolvm/csi-topolvm.sock"
+            - "--feature-gates=Topology=true"
+            - --enable-leader-election
+            - --leader-election-namespace={{ .Release.Namespace }}
+            - --leader-election-type=leases
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /run/topolvm
+
+        - name: csi-attacher
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
These images should probably be separate entries, so the upstream proxies can be used. They use different image names from each other.

Reply:
I'm sorry, I didn't fully understand